src/store-api/src/region_engine.rs

```diff
@@ -583,6 +583,60 @@ impl RegionStatistic {
     }
 }
 
+/// The response of syncing the manifest.
+#[derive(Debug)]
+pub enum SyncManifestResponse {
+    NotSupported,
+    Mito {
+        synced: bool,
+    },
+    Metric {
+        metadata_synced: bool,
+        data_synced: bool,
+        new_opened_logical_region_ids: Vec<RegionId>,
```

Comment:
nitpick: Better to document the relationship between `new_opened_logical_region_ids` and other fields.

Reply:
resolved in [cf4fc5a](https://github.com/GreptimeTeam/greptimedb/pull/5878/commits/cf4fc5a5cf4118988545c04892de6777b42272a6)